sickbeard/providers/torrentproject.py

```diff
@@ -101,6 +101,8 @@ def search(self, search_strings, age=0, ep_obj=None):  # pylint: disable=too-man
 
                     t_hash = torrents[i]["torrent_hash"]
                     torrent_size = torrents[i]["torrent_size"]
+                    if not all([t_hash, torrent_size]):
+                        continue
                     size = convert_size(torrent_size) or -1
```

Comment:
But you can if there is no size, right? Right now it will skip it if there is a hash, but no torrent_size.